website/pages/docs/agent/index.mdx

```diff
@@ -44,67 +38,67 @@ Please see the [Caching docs][caching] for information.
 
 These are the currently-available general configuration option:
 
-- `vault` <tt>([vault][vault]: \<optional\>)</tt> - Specifies the remote Vault server the Agent connects to.
+- `vault` <tt>([vault][vault]: <optional\>)</tt> - Specifies the remote Vault server the Agent connects to.
 
-- `auto_auth` <tt>([auto_auth][autoauth]: \<optional\>)</tt> - Specifies the method and other options used for Auto-Auth functionality.
+- `auto_auth` <tt>([auto_auth][autoauth]: <optional\>)</tt> - Specifies the method and other options used for Auto-Auth functionality.
 
-- `cache` <tt>([cache][caching]: \<optional\>)</tt> - Specifies options used for Caching functionality.
+- `cache` <tt>([cache][caching]: <optional\>)</tt> - Specifies options used for Caching functionality.
 
-- `listener` <tt>([listener][listener]: \<optional\>)</tt> - Specifies the addresses and ports on which the Agent will respond to requests.
+- `listener` <tt>([listener][listener]: <optional\>)</tt> - Specifies the addresses and ports on which the Agent will respond to requests.
 
-- `pid_file` `(string: "")` - Path to the file in which the agent's Process ID
+- `pid_file` <tt>(string: "")</tt> - Path to the file in which the agent's Process ID
```

Comment:
In other docs we typically just use tic marks instead of `<tt></tt>`:
```
`pid_file` `(string: "")` - Path to the file in which the agent's Process ID
```

Reply:
I think the `<tt></tt>` isn't working. Looking at the [preview](https://5eb453dea9c0260007f023be--vault-www.netlify.app/docs/agent#configuration) it isn't codifying `(string: "")`

<img width="403" alt="Screen Shot 2020-05-07 at 12 47 21 PM" src="https://user-images.githubusercontent.com/72365/81332835-e9ae5d00-9060-11ea-93ca-8c91ac6bc41b.png">